data/actions/scripts/other/create_bread.lua

```diff
@@ -4,7 +4,7 @@ local millstones = {1381, 1382, 1383, 1384}
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	local itemId = item:getId()
 	if itemId == 2692 then
-		if target:getSubType() == 1 and table.contains(liquidContainers, target:getId()) then
+		if target:getCount() == 1 and table.contains(liquidContainers, target:getId()) then
```

Comment:
here I think it is subType for checking for water

Reply:
getSubType()/getCount() does not work.
![image](https://user-images.githubusercontent.com/53269692/116491869-43f78d80-a871-11eb-906f-03591b741040.png)
![image](https://user-images.githubusercontent.com/53269692/116492110-d304a580-a871-11eb-86c2-25987f293a67.png)
Only `target.type` work.

Today I am going to review the scripts that I changed since the suggestion. It became a mess 👺

Reply:
yeah, I think we should keep the .type like it was before